crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -178,4 +187,25 @@ def enforce_installer
       end
     end
   end
+
+  def sanity_checks_cache
+    respond_to do |format|
+      format.html do
```

Comment:
So if the first thing we do for crowbar is a json formatted request, then we don't do the sanity checks?